docker-compose.yml

```diff
@@ -204,19 +206,19 @@ services:
     stdin_open: true
```

Comment:
These two lines (stdin_open and tty) were only there to force docker to rebuild the mfr/wb containers. They're no longer needed after the PRs Fitz merged yesterday. Plus, they just make your terminal break.